lib/enum_timestamp.ex

```diff
@@ -0,0 +1,26 @@
+defmodule EnumTimestamp do
```

Comment:
what do you think about renaming this to `CoursePlanner.Enum` or something else?  I don't think the focus should be on the timestamp, that's just some convenient information